packages/accordion/src/vaadin-accordion-panel.js

```diff
@@ -32,10 +50,87 @@ import { Details } from '@vaadin/details/src/vaadin-details.js';
  *
  * @fires {CustomEvent} opened-changed - Fired when the `opened` property changes.
  */
-class AccordionPanel extends Details {
+class AccordionPanel extends DetailsMixin(
+  DelegateFocusMixin(DelegateStateMixin(ThemableMixin(ControllerMixin(PolymerElement)))),
+) {
   static get is() {
     return 'vaadin-accordion-panel';
   }
+
+  static get template() {
+    return html`
+      <style>
+        :host {
+          display: block;
+        }
+
+        :host([hidden]) {
+          display: none !important;
+        }
+
+        [part='content'] {
+          display: none;
+          overflow: hidden;
```

Comment:
nit: I'd assume this isn't necessary since it already has `display: none`

Reply:
Removed and updated tests to only check for `display` property.